src/components/DonutChart/DonutChart.stories.mdx

```diff
@@ -0,0 +1,45 @@
+# DonutChart Component
+
+## Type: Simple
+
+## DonutChart Props
+
+|        Prop        |  Type  |                                                      Description                                                       | Default | Mandatory |
+| :----------------: | :----: | :--------------------------------------------------------------------------------------------------------------------: | :-----: | :-------: |
+|        size        | number | Specifies the overal size of the Donut Chart. We consider this number to calculate the chart radius and circunference. |    -    |   true    |
+|    strokewidth     | number |                        Specifies the width of the border you want to have on your donut chart.                         |    -    |   true    |
+|       value        | number |                        Receives the percentage of the filled portion we will show on the chart                         |    -    |   true    |
```

Comment:
instead of value, maybe we could replace it by `completion` given it is more specific to what we are using this prop for